DESCRIPTION

```diff
@@ -87,10 +87,12 @@ Suggests:
     dygraphs,
     tibble,
     fs,
-    callr (>= 2.0.0)
+    callr (>= 2.0.0),
+    rmarkdown
```

Comment:
Is it necessary for **rmarkdown** to suggest itself? I guess `R CMD check` won't complain if **rmarkdown** is not in its own `Suggests` when building vignettes.